### ISSUE TITLE
regularizer is not included in total loss

### ISSUE THREAD

**Issue by hgjung97** [pythonlessons/TensorFlow-2.x-YOLOv3#165]
The current train loop calculates the total loss as below:
total_loss = giou_loss + conf_loss + prob_loss

In this implementation doesn't include regularizer loss, such as kernel_regularizer of Conv2D.
In my opinion, the above sentence should be modified as below to reflect regularizers.
total_loss = giou_loss + conf_loss + prob_loss + tf.add_n(yolo.losses)

Reference: https://github.com/tensorflow/tensorflow/issues/21587

**Comment by pythonlessons** [comment]
Hmmm, this looks interesting, need to look at it

**Comment by pythonlessons** [comment]
> When I applied such a modification, I got different results.

What were the results, were they better, have you compared the mAP?
